crates/core/src/surfnet/svm.rs

```diff
@@ -712,6 +715,17 @@ impl SurfnetSvm {
                 slot,
                 None,
             );
+            let Some(SurfnetTransactionStatus::Processed(tx_data)) =
+                self.transactions.get(&signature)
+            else {
+                continue;
+            };
+            self.notify_logs_subscribers(
+                &signature,
+                None,
+                tx_data.meta.log_messages.clone().unwrap_or(vec![]),
+                CommitmentLevel::Confirmed,
+            );
```

Comment:
Do we need to notify when Finalized as well? (below in the `finalize_transactions` fn)

Reply:
Good catch, I had it in a local stash!